stdlib/queue.gr

```diff
@@ -137,6 +138,37 @@ provide let pop = queue => {
   }
 }
 
+/**
+ * Converts a queue into a list of its elements.
+ *
+ * @param queue: The queue to convert
+ * @returns A list containing all queue values
+ *
+ * @since v0.6.0
+ */
+provide let toList = queue => {
+  let lst = List.init(size(queue), i => match (queue.array[queue.headIndex +
```

Comment:
since you are using internals to implement this, why wouldn't you just use `queue.size` instead of calling the function?